components/experimental/src/units/provider.rs

```diff
@@ -42,6 +43,7 @@ use super::ratio::IcuRatio;
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 pub struct UnitsInfoV1<'data> {
+    // TODO: remove this field once we are using this map from `measure/provider::UnitsTrie`.
```

Comment:
does this mean with this PR there is data duplication?

Reply:
yes, exactly

Reply:
I don't like that you're adding so many TODOs in each PR. To me as a reviewer it's very hard to follow what is happening across multiple PRs, especially if I don't review all of them. I would prefer if you addressed the data duplication in this PR.

Reply:
The problem, if I finalized this PR with everything, it will be so huge that it is hard to be reviewed. I like to split the PRs to a reasonable amount of code.